test/rules/prefer-ignore-reason.test.js

```diff
@@ -0,0 +1,44 @@
+const {RuleTester} = require('eslint');
+const rule = require('../../rules/prefer-ignore-reason');
+
+const ruleTester = new RuleTester();
+
+ruleTester.run('prefer-ignore-reason', rule, {
+	valid: [
+		'/* istanbul ignore next: lazy to test */',
+		'/* istanbul ignore if: e2e */',
+		'// istanbul ignore next: with comment',
+		'// istanbul ignore function: with comment',
```

Comment:
```suggestion
		'// istanbul ignore file: with comment',
```

There is no `istanbul ignore function` (that is accomplished with `istanbul ignore next` before the function).